nncf/common/accuracy_aware_training/training_loop.py

```diff
@@ -62,6 +62,13 @@ def run(self, model: TModel, train_epoch_fn, validate_fn, configure_optimizers_f
         :return: The fine-tuned model
         """
 
+    @property
+    @abstractmethod
+    def final_statistics(self):
```

Comment:
At some point I had one property for non-final and final statistics, but then decided to separate them due to implementation complications and because only the `final_statistics` is really needed by the user. 

I have changed it now to a single property. Seems better.